# DO NOT EDIT. RUN THE PYTHON FILE INSTEAD TO GENERATE THIS .yaml FILE 
# date: 2023-04-08
traj_x: 0.333333333333333*sin(0.666666666666667*t) + 0.666666666666667*sin(1.33333333333333*t)
traj_y: 0.333333333333333*cos(0.666666666666667*t) - 0.666666666666667*cos(1.33333333333333*t)
traj_z: 1.5 - 0.5*sin(2.0*t)
bbox: [0.6, 0.6, 0.2]
